### ISSUE TITLE
[Masthead] With custom data - After activating back from the sub menu options,  screen reader users are not able to navigate using left and right swipe

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5148]
**Engineering info:**  
- Innovation Team engineer: Pending Innovation team's 02/21 sprint planning
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2417
- DPO consulting engineer:

-----

## Detailed description

On Voiceover iPhone on Safari
Issue:
After activation the sub options are collapsed visually but screen reader users are not able to navigate using left and right swipe.
screen reader announces  only back.

1. On Safari, go to first menu item and tap into submenu
2. tap on the back button
3. try to swipe left and right

Expected:
after activation screen reader users should be able to use left and right swipe.

**Comment by annawen1** [comment]
@Praveen-sr can you retest this? I'm not able to reproduce this on my device.

**Comment by andy-blum** [comment]
This issue seems to have been resolved when we fixed the focus trap and keyboard nav issues within the last several months

https://user-images.githubusercontent.com/25532785/155207801-01e88aa8-2a67-448c-9fe9-3323f428c129.mp4
